pegasus/sites.v3/code.org/views/gen_ai_landing_page/faq_generative_ai.haml

```diff
@@ -0,0 +1,33 @@
+%details
+  %summary=hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.question")
+  %p=hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.answer.overview", markdown: :inline, locals: {essential_literacy_url: "https://www.teachai.org/media/what-is-ai?page=%2Fpolicy&contentGrid=widget_P4nnc1XII"})
+  %ul
+    %li
+      %strong
+        =hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.answer.full_course.label")
+      =hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.answer.full_course.desc")
+    %li
+      %strong
+        =hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.answer.individual_units.label")
+      =hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.answer.individual_units.desc")
+  %p=hoc_s("gen_ai_page.faq.how_to_fit_existing_curriculum.answer.conclusion")
+%details
+  %summary=hoc_s("gen_ai_page.faq.address_ethical_considerations.question")
+  %p=hoc_s("gen_ai_page.faq.address_ethical_considerations.answer", markdown: :inline, locals: {teachers_guide_url: "https://docs.google.com/document/u/0/d/1ztq07VnPY-KfuA1dQEeHjCa5yd4Y-E6qc9RdUIl6vDI/edit"})
+%details
+  %summary=hoc_s("gen_ai_page.faq.tools_and_setup.question")
+  %p=hoc_s("gen_ai_page.faq.tools_and_setup.answer", markdown: :inline, locals: {it_requirements_url: "/educate/it", technical_support_url: "https://codeorg.zendesk.com/knowledge/editor/01J7GRRVJGMZT4SRAFY9AED99M/en-us?brand_id=1346"})
```

Comment:
Let's leave this blank until we get the live link. I left a [comment](https://www.figma.com/design/cnqwFBmgKQkMJrDMiADIrW?node-id=2307-2&m=dev#950889222) on the Figma 
```suggestion
  %p=hoc_s("gen_ai_page.faq.tools_and_setup.answer", markdown: :inline, locals: {it_requirements_url: "/educate/it", technical_support_url: "#"})
```